main_app/templates/home.html

```diff
@@ -3,36 +3,70 @@
 {% block content %}
 
 {% if user.is_authenticated %}
-<div class="col-md-6">
-  <h1>Welcome to IVM</h1>
-</div>
+<h3 class="m-4">All Items in your inventory</h3>
+<div class="row">
+  <div class="col-12">
+    {% if items %}
+    <div class="card bg-light ms-4 me-4 mb-4">
+      <div class="card-header">
+        <i class="fa-solid fa-list fa-lg"></i> Item Records
+      </div>
+      <div class="card-body">
+        <p class="card-text">
+          <div class="table-responsive ">
+            <table class="table table-hover">
+              <thead>
+                <tr>
+                    <th scope="col">ID</th>
+                    <th scope="col">Name</th>
+                    <th scope="col">Type</th>
+                    <th scope="col">Description</th>
+                    <th scope="col">Quantity</th>
+                    <th scope="col">Max. Quantity</th>
+                    <th scope="col">View</th>
+                    <th scope="col">Edit</th>
+                    <th scope="col">Delete</th>
+                </tr>
+              </thead>
+              <tbody>
+                {% for item in items %}
+                  <tr>
+                    <td>{{ item.id }}</td>
+                    <td>{{ item.name }}</td>
+                    <td>{{ item.type }}</td>
+                    <td>{{ item.description }}</td>
+                    <td>{{ item.quantity_current }}</td>
+                    <td>{{ item.quantity_max }}</td>
+                    <td>
+                        <a class="btn btn btn-info" href="{% url 'item_update' item.id %}">
+                          <i class="fa-solid fa-eye fa-lg"></i>
+                        </a>
+                      </td>
+                    <td>
+                      <a class="btn btn-warning me-1" href="{% url 'item_update' item.id %}">
```

Comment:
What's going on here with the warning button? I didn't see any extra button in the screenshot you showed us earlier today.